### PR TITLE
Add lang support at messages without it

### PR DESCRIPTION
**Pull request by sergeybochkarev**
And move language context from `app` to `modcms`

### CODE REVIEW

File: actions/PageAction.php [hunk 1]
```diff
@@ -60,7 +60,7 @@ public function run()
                 ]);
             }
         }
-        throw new NotFoundHttpException('The requested page does not exist.');
+        throw new NotFoundHttpException(App::t('modcms', 'The requested page does not exist.'));
```
Comment (disem) [inline actions/PageAction.php:63]: What is `App::t`?

File: README.md [hunk 2]
```diff
@@ -3,6 +3,12 @@ CMS Extension
 * [![Scrutinizer Code Quality](https://scrutinizer-ci.com/g/yii2mod/yii2-cms/badges/quality-score.png?b=master)](https://scrutinizer-ci.com/g/yii2mod/yii2-cms/?branch=master) 
 * [![Build Status](https://scrutinizer-ci.com/g/yii2mod/yii2-cms/badges/build.png?b=master)](https://scrutinizer-ci.com/g/yii2mod/yii2-cms/build-status/master)
 
+Update warning
+--------------
+
+Warning!
+Messages moved from `app` to `modcms`. Beware.
```
Comment (disem) [inline README.md:10]: Let's use similar to the extension namespace i18n category like `yii2mod.cms` ?

File: views/cms/_form.php [hunk 3]
```diff
@@ -73,13 +73,16 @@
 
     <?php echo $form->field($model, 'metaKeywords')->textarea(['rows' => 6]); ?>
 
-    <?php echo $form->field($model, 'status')->dropDownList(CmsStatus::$list); ?>
+    <?php echo $form->field($model, 'status')->dropDownList([
+        Yii::t('modcms', CmsStatus::$$list[CmsStatus::ENABLED]),
+        Yii::t('modcms', CmsStatus::$$list[CmsStatus::DISABLED]),
+    ]); ?>
```
Comment (disem) [inline views/cms/_form.php:79]: We should use listData from BaseEnum https://github.com/yii2mod/yii2-enum/blob/master/helpers/BaseEnum.php#L129
I will update it soon to allow adjustments for message categories but you can use `yii2mod.enum` for now.

File: README.md (hunk 2 above)
Comment (sergeybochkarev) [inline README.md:10]: Ok. Sorry, don't see it before.

**Pull request closed without merging**